app/src/main/java/ar/com/wolox/android/example/ui/home/news/NewsFragment.kt

```diff
@@ -1,15 +1,63 @@
 package ar.com.wolox.android.example.ui.home.news
 
+import android.content.Intent
+import android.widget.Toast
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.RecyclerView
 import ar.com.wolox.android.R
+import ar.com.wolox.android.example.ui.home.news.newsCreation.NewsCreationActivity
 import ar.com.wolox.wolmo.core.fragment.WolmoFragment
+import com.facebook.drawee.backends.pipeline.Fresco
+import kotlinx.android.synthetic.main.fragment_news.*
 import javax.inject.Inject
 
 class NewsFragment @Inject constructor() : WolmoFragment<NewsPresenter>(), INewsView {
 
+    private lateinit var mNewsDataList: ArrayList<String>
+    private lateinit var mViewAdapter: RecyclerView.Adapter<*>
+    private lateinit var mViewManager: RecyclerView.LayoutManager
+
     override fun layout(): Int {
         return R.layout.fragment_news
     }
 
     override fun init() {
+        Fresco.initialize(context)
+        mNewsDataList = ArrayList()
+
+        addContacts()
+
+        mViewManager = LinearLayoutManager(context)
+        mViewAdapter = NewsDataAdapter(mNewsDataList)
+
+        vRecyclerViewNews.apply {
+            setHasFixedSize(true)
+            layoutManager = mViewManager
+            adapter = mViewAdapter
+        }
+
+        fab_icon.setOnClickListener {
+            presenter.addNewsPressed()
+        }
+
+        vSwipeRefreshLayout.setOnRefreshListener {
+            presenter.onRefresh()
+        }
+    }
+
+    private fun addContacts() {
+        for (i in 0..9) {
```

Comment:
When you do this temporal stuff just for showing a design, it would be reaaaally good to put a comment like "This will be removed when the actual news from the server is obtained".